app/models/form/sales/questions/property_number_of_bedrooms.rb

```diff
@@ -0,0 +1,12 @@
+class Form::Sales::Questions::PropertyNumberOfBedrooms < ::Form::Question
```

Comment:
This should be This should be `Form::Sales::Property::Questions::PropertyNumberOfBedrooms`